specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -6402,6 +6446,34 @@
         }
       }
     },
+    "InvoiceRebillDetails": {
+      "description": "The properties of a payment.",
+      "properties": {
+        "rebillDocumentType": {
+          "description": "The document type of the invoice in a Enterprise Agreement rebilled invoice relationship.",
+          "type": "string",
+          "enum": [
+            "Credit",
+            "Rebill",
+            "Original"
+          ],
+          "x-ms-enum": {
+            "name": "rebillDocumentType",
+            "modelAsString": true
+          }
+        },
+        "rebilledInvoiceId": {
+          "description": "The ID of the rebilled invoice in a Enterprise Agreement rebilled invoice relationship",
+          "type": "string",
+          "readOnly": true
+        },
+        "latestInvoiceId": {
+          "description": "The ID of the latest invoice in a Enterprise Agreement rebilled invoice relationship.",
```

Comment:
The ID of the latest invoice generated when an Enterprise Agreement invoice is rebilled,

Reply:
This is not specific to the multiple PDF per invoice scenario.  This is for rebill scenarios.  It references the latest generated invoice.  In a rebill scenario, there is a sequence of invoices, where the earliest is the originally-generated invoice, and the latest is the most recently generated rebill invoice.  The latest rebill invoice is the one that would be shown in the invoice list experience, and all of the earlier invoices in the sequence would be shown in the Rebill Details view for that one.